### ISSUE TITLE
Parametrisation file not used

### ISSUE THREAD

**Issue by hoelzerC** [grimme-lab/xtb#822]
**Describe the bug**
The parametrization files given in `./xtb` are not used in the binary. 

**To Reproduce**
Steps to reproduce the behaviour:

1. Change parametrization in e.g. `./xtb/param_gfn1-xtb.txt`
2. Compile `xtb`
3. Run `xtb --gfn1 random.xyz`
4. Results in identical output, as before, i.e. change of parametrization has no effect.

**Expected behaviour**
Since the `./xtb/param_gfn*.txt` files are required during compilation by meson, one would expect these to contain the parametrization used in the compiled binary. 

**Additional context**
Apparently, this issue can be circumferences by moving the edited parametrization file into your home directory `~/`. However this is not a sustainable solution an fails for shipping `xtb` binary to external clusters, sharing with collaborators etc.

**Comment by hoelzerC** [comment]
As discussed with @Albkat

**Comment by hoelzerC** [comment]
@awvwgk Yes, absolutely! We are aware of that option, which is very handy for individual on-the-fly `xtb` calculations. (Although the documentation for the `--vparam `and `--xparam` flags could be improved.) However, for adapting `xtb` in larger frameworks such as `CREST`, a compiled version using a given set of parameters is necessary.

**Comment by MtoLStoN** [comment]
Thank you, @awvwgk!

I do not believe that changing default parameters in a pre-built binary is the way to go, and I would prefer not to do that, as it increases the bug potential to ship several "stable" xTB versions. As Sebastian said, we already have several ways to change the default parametrization in a pre-built binary by providing a custom parameter file (@awvwgk, I think you probably meant the ``XTBHOME`` variable, just for future reference). If we want to ship a custom parametrization, we can quickly do so. Nonetheless, if one wants to change the (hard-coded) default parametrization, this can be done in the code itself. 

However, this should probably be better documented.

**Comment by awvwgk** [comment]
`XTBHOME` has been deprecated since the initial 6.x release and is only supported for backwards compatibility. Please use `XTBPATH` instead.

**Comment by MtoLStoN** [comment]
Oops, you are right of course; thanks for clarifying.

**Comment by hoelzerC** [comment]
`If we want to ship a custom parametrization, we can quickly do so.`
How can this be done in the context of `CREST` for example? Changing global variables does not solve this appropriately e.g. for usage on servers.

**Comment by hoelzerC** [comment]
To summarize the current status: 
the `param*.txt` files are in the `src` binary, needed for compilation, but the values are taken from the fortran source code. Changing the values in the `param*.txt` files does not change the compiled binary. This is also not desired, since you are supposed to set the global variable `XTBPATH` correctly.

Since external people want to use `xtb` with different parameterizations, the workflow described above should be streamlined for ease of use. As the behavior of the parameterizations is not clear without deeper consultation of the source code, at least the documentation should be adapted in such a way that the interactive customization of the parameterization (via `--vparam`) and the static customization (via `XTBPATH`) are clearly visible.   

The [previous documentation](https://xtb-docs.readthedocs.io/en/latest/setup.html?highlight=xtbpath#environment-variables-for-xtb)  is a bit sparse on the details about it.

Also [this statement](https://xtb-docs.readthedocs.io/en/latest/sp.html?highlight=vparam#h0-tuning) is wrong then: 

> Warning
> Please do not change the original param_gfn2-xtb.txt file. Otherwise, global parameters are changed.

**Comment by MtoLStoN** [comment]
I completely agree that there should be better documentation, particularly if the custom parameterization of xTB will become a more widely used option. It would be best to create a new section in the documentation dedicated to this topic.
